src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
+                    },
+                    {
+                      label: 'Inactive',                        
+                      value: 'inactive',
+                      selected: false,
+                    },
+                  ],
+                  advanced: true,
+                  value: 'active',
+                },
+                {
+                  name: 'inputAdvanced',
+                  label: 'Label',
+                  id: 'myoption5',
+                  advanced: true,
+                },
+                {
+                  name: 'textareaAdvanced',
+                  label: 'textarea',
+                  id: 'myoption6',
+                  type: 'textarea',
+                  advanced: true,
+                },
+                {
+                  name: 'checkboxAdvanced',
+                  id: 'myoption7',
+                  label: 'Advanced Checkbox',
+                  type: 'checkbox',                    
+                  advanced: true,
+                },
+              ],
+              columnsData: {
+                columns: [
+                  {
+                    name: 'columnA',
+                    label: 'Column A',
+                  },
+                  {
+                    name: 'columnB',
+                    label: 'Column B',
+                  },
+                  {
+                    name: 'columnC',
+                    label: 'Column C',
+                    locked: true,
+                    selected: true,
+                  },
+                  {
+                    name: 'columnD',
+                    label: 'Column D',
+                    locked: true,
+                    selected: true,
+                  },
+                  {
+                    name: 'columnE',
+                    label: 'Column E',
+                    selected: true,
+                  },
+                  {
+                    name: 'columnF',
+                    label: 'Column F',
+                    selected: true,
+                  },
+                  {
+                    name: 'columnG',
+                    label: 'Column G',
+                  },
+                  {
+                    name: 'columnH',
+                    label: 'Column H',
+                  },
+                  {
+                    name: 'columnI',
+                    label: 'Column I',
+                  },
+                  {
+                    name: 'columnJ',
+                    label: 'Column J',
+                  },
+                ],
+              }
+            };
+          };
+        };
+    .chi-tabs-panel#html-portal
+      .chi-tab__description
+        p.-text
+          | To render toolbar, use the class <code>chi-toolbar</code>.
+        span.-text--grey
+          | Use the slots <code>start</code> to add elements to the left side and the <code>end</code> to add the elements to the right  side of Toolbar header.
+      :code(lang='html')
+        <div class="chi-toolbar -portal">
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.